Lib/dis.py

```diff
@@ -32,20 +32,30 @@ def _try_compile(source, name):
     return c
 
 def dis(x=None, *, file=None, depth=None):
-    """Disassemble classes, methods, functions, generators, or code.
+    """Disassemble classes, methods, functions, and other compiled objects.
 
-    With no argument, disassemble the last traceback.
+       With no argument, disassemble the last traceback.
```

Comment:
We *are* sometimes inconsistent, but as @serhiy-storchaka notes, the preferred layouts are the ones that align with the opening triple-quote - the last option isn't shown in PEP 257 because we're not supposed to use it :)

The inconsistent ones just get left alone once they're in, since we don't typically do formatting-only edits - we're more likely to fix the indentation as part of a change that needed to update the affected docstring anyway.